### PR TITLE
Schema diagram to database schema

### PR DESCRIPTION
**Pull request by Gbengacode**
### We made the following changes to this work

- Created the patient's table
- Created the medical histories table
- Created the invoices table and added a foreign key
- Created the treatments table and added a foreign key
- Created the invoice_items table and added foreign keys
-  Created indexed for the foreign keys
- Created the many-to-many relationship table
- Updated the README file

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+);
+
+DROP TABLE IF EXISTS treatments;
+
+CREATE TABLE treatments (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  type VARCHAR(50) NOT NULL,
+  name VARCHAR(50) NOT NULL,
+);
```
Comment (julie-ify) [inline schema_based_on_diagram.sql]: - [ ] According to the ERD diagram, there are five(5) visible tables and one(1) join table to be created making it six(6) tables in total to be created. However, I can see that you created only four(4) tables. Please let's follow the diagram and ensure that all the tables are created so as to match the diagram provided to you. 👍

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  ON medical_histories
+  (patient_id);
+
+CREATE INDEX invoices_id_index 
+  ON invoices_items
+  (invoices_id);
+
+CREATE INDEX treatment_id_index 
+  ON invoices_items
+ (treatment_id);
```
Comment (julie-ify) [inline schema_based_on_diagram.sql]: - [ ] Let's ensure that we follow the ERD by making sure that the names of the tables in the diagram provided are the same as yours. Please consider changing this `invoices_items` to `invoice_items` and also the column `invoices_id` to `invoice_id` so as to match the ERD provided 👍.

File: schema_based_on_diagram.sql [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+DROP TABLE IF EXISTS treatments_medical_histories;
+
+CREATE TABLE treatments_medical_histories (
+  treatment_id INT NOT NULL,
+  medical_history_id INT NOT NULL,
+  FOREIGN KEY (treatment_id) REFERENCES treatments (id),
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id)
+
+);
```
Comment (julie-ify) [inline schema_based_on_diagram.sql]: - [ ] Let's make sure that all tables have a PRIMARY KEY set. Please add a primary key to this table 💯

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Gbengacode and @youmari,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @Gbengacode,


## Wow, you did it 🎉
Brilliant
## Thank you for creating the missing tables, correcting the column names, and adding the missing primary key as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**